app/archivesspace/form.html

```diff
@@ -1,8 +1,35 @@
 <form class="resource-form">
-  Title: <input type="text" ng-model="form.title">
-  Level of description: <select ng-model="form.level"
-                                ng-options="level for level in form.levels">
-  </select>
-  <input type="submit" ng-click="form.ok()" value="Save" />
-  <input type="submit" ng-click="form.cancel()" value="Cancel" />
+  <div class="row">
+    <div class="col-md-6">
+      <label>Title:</label> <input type="text" ng-model="form.title">
+    </div>
+    <div class="col-md-6">
+      <label>Level of description:</label> <select ng-model="form.level"
+                                  ng-options="level for level in form.levels">
+                                  </select>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-md-6">
+      <label>Note:</label> <textarea ng-model="form.note"></textarea>
+    </div>
+  </div>
+  <!-- Start and end dates are only in the form for creating a new item, not editing -->
+  <div class="row" ng-if="form.start_date !== false &amp;&amp; form.end_date !== false">
```

Comment:
It is, yeah. Since this is in HTML, it evaluates to `&&` and is considered to be `&&` by Angular.